src/esp/assets/ResourceManager.cpp

```diff
@@ -349,9 +349,17 @@ int ResourceManager::loadObject(const std::string objPhysConfigFilename,
     MeshMetaData meshMetaData = resourceDict_[filename];
     scene::SceneNode& newNode = parent->createChild();
     AssetInfo renderMeshinfo = AssetInfo::fromPath(filename);
-
+    Magnum::PluginManager::Manager<Importer> manager{
+#ifdef MAGNUM_BUILD_STATIC
+        "./"
+#endif
```

Comment:
FYI, this `#ifdef` is not needed since https://github.com/mosra/corrade/commit/1329656464ed4afbeaa1ff96ad0f0f5ec3149a0f, simply instantiating the manager is enough. Same below :)